src/ipred.rs

```diff
@@ -2007,12 +1995,15 @@ mod neon {
         max_width: c_int,
         max_height: c_int,
         bitdepth_max: c_int,
-        _topleft_off: usize,
+        topleft_off: usize,
     ) {
+        // SAFETY: Reconstructed from args passed by `angular_ipred::Fn::call`.
```

Comment:
Can you also add the `# Safety` comment on `fn ipred_z_neon_erased` saying it must be called by ``[`angular_ipred::Fn::call`]``?